quantized_mesh_tile/bbsphere.py

```diff
@@ -25,7 +33,7 @@ def fromPoints(self, points):
             raise Exception('Your list of points must contain at least 2 points')
 
         nbPositions = len(points)
-        for i in xrange(0, nbPositions):
+        for i in range(0, nbPositions):
```

Comment:
I'd prefer to keep a generator in python 2.7, maybe:
`from past.builtins import xrange`
this way we make sure we always use generators